scripts/set_vars.sh

```diff
@@ -8,8 +8,8 @@ export PLATFORM_PATH="${PLATFORM_PATH:-${ROOT_PATH}}"
 export CONFIG_PATH="${CONFIG_PATH:-${PLATFORM_PATH}/ostis-web-platform.ini}"
 export REPO_PATH="${REPO_PATH:-${PLATFORM_PATH}/repo.path}"
 
-export SC_MACHINE_REPO="${SC_MACHINE_REPO:-https://github.com/ostis-ai/sc-machine.git}"
-export SC_MACHINE_BRANCH="${SC_MACHINE_BRANCH:-main}"
+export SC_MACHINE_REPO="${SC_MACHINE_REPO:-https://github.com/NikitaZotov/sc-machine.git}"
```

Comment:
I think we shouldn't merge custom repo URL to `develop`

Reply:
> I think we shouldn't merge custom repo URL to `develop`

There is separated commit for this to test ci